airbyte-container-orchestrator/build.gradle

```diff
@@ -54,6 +57,18 @@ application {
     applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0']
 }
 
+task generateWellKnownTypes() {
```

Comment:
let's put a comment here saying it's copy-pasted from the worker build.gradle. At some point I want to move this generation into airbyte-protocol directly (and just have it dump the yaml into `build/whatever` so that all modules can just use it directly).

Reply:
updated